### PR TITLE
feat: navigation and popular products

### PR DESCRIPTION
**Pull request by AmishMishra11**
https://user-images.githubusercontent.com/76205249/158802155-576dddb5-ba55-4cfa-bcaf-b97a1f939fe3.mp4

-Added Popular products to be displayed using mockbee backend
-Added Navigation

Home Page finished.

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
I hope you will work on your color choices later!

File: src/Components/Footer/styles.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  flex: 1 1 0px;
+  justify-content: space-around;
+}
+.footer-main li {
+  margin: 1rem 0;
+}
+
+.footer-main li a {
+  color: white;
+}
```
Comment (prathmeshjagtap) [inline src/Components/Footer/styles.css:21]: you can use variable naming for color

File: src/Components/Popular/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+function Popular() {
+  const [popular, setPopular] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get("/api/products");
+        if (res.status === 200) {
+          const products = res.data.products;
+          console.log(products);
```
Comment (prathmeshjagtap) [inline src/Components/Popular/index.js]: remove console log

**Review by prathmeshjagtap — CHANGES_REQUESTED** [review]

File: src/Components/Footer/styles.css (hunk 1 above)
Comment (AmishMishra11) [reply src/Components/Footer/styles.css:21]: I use them but only for black and white I use normal color names

File: src/Components/Popular/index.js (hunk 2 above)
Comment (AmishMishra11) [reply src/Components/Popular/index.js]: Thank you for pointing it out i will remove them

**Pull request merged**